### PR TITLE
Dev piotrek hook

### PR DESCRIPTION
**Pull request by sabath91**

### CODE REVIEW

File: scripts/clone.sh [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
 # Have our remote git repository (https://github.com/korotkevics/ships/)
-# cloned into a [your_dir_name].
+# cloned into a [your_dir_name] and set pre-commit hook to run mvn clean install.
 #
 
 set -e
 git clone https://github.com/korotkevics/ships $1 --quiet
 cd $1
 git checkout master --quiet
+
```
Comment (korotkevics) [inline scripts/clone.sh]: Nope, please create a separate script which only uses clone.sh, as we use clone.sh in other scripts

File: scripts/pre-commitHookMaven.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+# retrieving directory of this script
+MAIN_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+echo  $MAIN_DIR
+
+# go to main project dir
+cd $MAIN_DIR/../../
+
+# running maven clean install
+mvn clean install
```
Comment (korotkevics) [inline scripts/pre-commitHookMaven.sh]: please add -q to silence it

Comment (sabath91) [reply scripts/pre-commitHookMaven.sh]: done

File: scripts/clone.sh (hunk 1 above)
Comment (sabath91) [reply scripts/clone.sh]: done

**Pull request merged**